app/apptesting/test_suite.go

```diff
@@ -65,18 +65,18 @@ func (keeperTestHelper *KeeperTestHelper) SetupValidator(bondStatus stakingtypes
 	return valAddr
 }
 
-func (keeperTestHelper *KeeperTestHelper) BeginNewBlock(executeNextEpoch bool) {
+func (keeperTestHelper *KeeperTestHelper) BeginNewBlock(executeNextEpoch bool, proposer sdk.ValAddress) {
```

Comment:
Perhaps we should make a `BeginNewBlock` and a `BeginNewBlockWithProposer` functions? I imagine in a lot of cases, we don't want to deal with the caller choosing the proposer